### ISSUE TITLE
Panel Not Displaying Any Data

### ISSUE THREAD

**Issue by Scott-St** [NatelEnergy/grafana-discrete-panel#82]
This just started in the past month or so but the panel is no longer displaying any data.  When I click edit or view it does nothing as well.  

I tried to delete the panel plugin folder but it still didnt seem to fix it.  Restarted grafana multiple times.  This is the only panel plugin that doesnt seem to be working.

The log indicates it was downloaded and isntalled successfully so I am not really sure what else I can look for to try.

```
installing grafana-piechart-panel @ 1.3.4
from url: https://grafana.com/api/plugins/grafana-piechart-panel/versions/1.3.4/download
into: /var/lib/grafana/plugins

✔ Installed grafana-piechart-panel successfully

Restart grafana after installing plugins . <service grafana-server restart>

installing natel-influx-admin-panel @ 0.0.5
from url: https://grafana.com/api/plugins/natel-influx-admin-panel/versions/0.0.5/download
into: /var/lib/grafana/plugins

✔ Installed natel-influx-admin-panel successfully

Restart grafana after installing plugins . <service grafana-server restart>

installing ryantxu-ajax-panel @ 0.0.5
from url: https://grafana.com/api/plugins/ryantxu-ajax-panel/versions/0.0.5/download
into: /var/lib/grafana/plugins

✔ Installed ryantxu-ajax-panel successfully

Restart grafana after installing plugins . <service grafana-server restart>

installing natel-discrete-panel @ 0.0.9
from url: https://grafana.com/api/plugins/natel-discrete-panel/versions/0.0.9/download
into: /var/lib/grafana/plugins

✔ Installed natel-discrete-panel successfully

Restart grafana after installing plugins . <service grafana-server restart>

installing natel-plotly-panel @ 0.0.5
from url: https://grafana.com/api/plugins/natel-plotly-panel/versions/0.0.5/download
into: /var/lib/grafana/plugins

✔ Installed natel-plotly-panel successfully

Restart grafana after installing plugins . <service grafana-server restart>

t=2019-01-09T10:02:26-0500 lvl=info msg="Starting Grafana" logger=server version=4.5.2 commit=ec2b0fe compiled=2017-09-22T04:34:03-0400

t=2019-01-09T10:02:26-0500 lvl=info msg="Config loaded from" logger=settings file=/usr/share/grafana/conf/defaults.ini
t=2019-01-09T10:02:26-0500 lvl=info msg="Config loaded from" logger=settings file=/etc/grafana/grafana.ini
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from command line" logger=settings arg="default.paths.data=/var/lib/grafana"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from command line" logger=settings arg="default.paths.logs=/var/log/grafana"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from command line" logger=settings arg="default.paths.plugins=/var/lib/grafana/plugins"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from command line" logger=settings arg="default.log.mode=console"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_SERVER_DOMAIN=<REMOVED>"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_SERVER_ROOT_URL=https://<REMOVED>"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_SESSION_PROVIDER=file"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_ANALYTICS_REPORTING_ENABLED=false"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_ANALYTICS_CHECK_FOR_UPDATES=false"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_SECURITY_ADMIN_PASSWORD=*********"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_USERS_ALLOW_SIGN_UP=false"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_AUTH_ANONYMOUS_ENABLED=false"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_AUTH_BASIC_ENABLED=false"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_AUTH_PROXY_ENABLED=true"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_AUTH_PROXY_HEADER_NAME=X-WEBAUTH-USERNAME"
t=2019-01-09T10:02:26-0500 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_LOG_LEVEL=debug"
t=2019-01-09T10:02:26-0500 lvl=info msg="Path Home" logger=settings path=/usr/share/grafana
t=2019-01-09T10:02:26-0500 lvl=info msg="Path Data" logger=settings path=/var/lib/grafana
t=2019-01-09T10:02:26-0500 lvl=info msg="Path Logs" logger=settings path=/var/log/grafana
t=2019-01-09T10:02:26-0500 lvl=info msg="Path Plugins" logger=settings path=/var/lib/grafana/plugins
t=2019-01-09T10:02:26-0500 lvl=info msg="Initializing DB" logger=sqlstore dbtype=sqlite3
t=2019-01-09T10:02:26-0500 lvl=info msg="Starting DB migration" logger=migrator

```

Tried in both ie and Chrome with the same result.

**Comment by Scott-St** [comment]
Actually, it looks like javascript errors.  This is the output of the chrome console.

[1547047194811.log](https://github.com/NatelEnergy/grafana-discrete-panel/files/2741437/grafana.stevensononthe.net-1547047194811.log)

**Comment by rdubya16** [comment]
Seeing this as well, getting the same TypeError

**Comment by bbczeuz** [comment]
Same issue here. Workaround: Use version 0.0.8-pre instead of 0.0.9 the new version doesn't bring any significant improvements to us.
Tested: Firefox 64 on Win10, Grafana 4.5.2 (we can't update because the grouped histogram graph is broken in newer releases...)

**Comment by ryantxu** [comment]
What versions are you guys using?  In grafana 5.4.0 the `panel-initialized` callback was not called, but that was fixed in 5.4.1+
https://github.com/grafana/grafana/issues/14287

**Comment by Scott-St** [comment]
Grafana v4.5.2
